### ISSUE TITLE
Backup-BcContainerDatabases doesn't work with external SQL Server

### ISSUE THREAD

**Issue by mkilinskidev** [microsoft/navcontainerhelper#2575]
Hi. I've noticed that if I have a database on the external SQL Server, the `Backup-BcContainerDatabases` prompt is trying to create a backup on the external SQL server which is obvious, but the path `C:\ProgramData\BcContainerHelper\Extensions\test1\` doesn't exist. When I'll create this path manually, everything works well. Is it possible to create this path remotely?

```
PS C:\Windows\system32> Backup-BcContainerDatabases -containerName test1 -databasecredential (New-Object pscredential 'user', (ConvertTo-SecureString 'password' -AsPlainText -Force))
Backing up test1-Cronus to C:\ProgramData\BcContainerHelper\Extensions\test1\database.bak
System.Data.SqlClient.SqlError: Cannot open backup device 'C:\ProgramData\BcContainerHelper\Extensions\test1\database.bak'. Operating system error 3(Sytem cannot find the path.).

Exception Script Stack Trace:
at Backup, <No file>: line 20
at <ScriptBlock>, <No file>: line 61

PowerShell Call Stack:
at Invoke-ScriptInBcContainer, C:\Program Files\WindowsPowerShell\Modules\BcContainerHelper\3.0.11\ContainerHandling\Invoke-ScriptInNavContainer.ps1: line 71                                                                                   at Backup-BcContainerDatabases, C:\Program Files\WindowsPowerShell\Modules\BcContainerHelper\3.0.11\Bacpac\Backup-NavContainerDatabases.ps1: line 52                                                                                            at <ScriptBlock>, <No file>: line 1                                                                                     
Container Free Physical Memory: 5.8Gb

Services in container test1:
- MicrosoftDynamicsNavServer$BC is Running
- MSSQL$SQLEXPRESS is Stopped

Backup-BcContainerDatabases Telemetry Correlation Id: bf51d04c-9034-48b0-a529-87301a744055
System.Data.SqlClient.SqlError: Cannot open backup device
'C:\ProgramData\BcContainerHelper\Extensions\test1\database.bak'. Operating system error 3(Sytem cannot find the path.).
At C:\Program
Files\WindowsPowerShell\Modules\BcContainerHelper\3.0.11\ContainerHandling\Invoke-ScriptInNavContainer.ps1:112 char:13
+             throw $errorMessage
+             ~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (System.Data.Sql...onej ścieżki.).:String) [], RuntimeException
    + FullyQualifiedErrorId : System.Data.SqlClient.SqlError: Cannot open backup device 'C:\ProgramData\BcContainerHel
   per\Extensions\test1\database.bak'. Operating system error 3(Sytem cannot find the path.).
```

**Comment by freddydk** [comment]
The issue here, seems to be that the containerhelperfolder is different from the hosthelperfolder.
Will be fixed in the next version

**Comment by freddydk** [comment]
fixed in the latest preview

**Comment by freddydk** [comment]
Shipped in 4.0.3
